css/custom.css

```diff
@@ -262,104 +260,115 @@ p.person-name {
   opacity: 1;
   visibility: visible;
 }
-#img_project{
+#img_project {
   display: none;
 }
-#map{ 
+#map {
   height: 600px;
   max-width: 1000px;
   margin: auto 7.5%;
   pointer-events: none;
 }
-.student_name{
+.student_name {
   font-size: 16px;
   margin-top: 5px;
   font-weight: bold;
 }
-.team_member_name{
+.team_member_name {
   font-size: 18px;
   margin-top: 5px;
   font-weight: bold;
   padding-bottom: 20px;
 }
-.team_member_designation{
+.team_member_designation {
```

Comment:
You are not supposed to Re-indent entire file when you commit. Only indent changes that you've done.